builder/config_reader_test.go

```diff
@@ -108,6 +109,33 @@ func testConfig(t *testing.T, when spec.G, it spec.S) {
 					h.AssertSliceContainsOnly(t, warns, "empty 'order' definition")
 				})
 			})
+
+			when("'uri' is misspelled as url", func() {
+				it.Before(func() {
+					h.AssertNil(t, ioutil.WriteFile(builderConfigPath, []byte(`
+[buildpack]
+url = "noop-buildpack.tgz"
+`), 0666))
+				})
+
+				it("returns errors", func() {
+					_, _, err := builder.ReadConfig(builderConfigPath)
+					h.AssertError(t, err, "parse contents of")
```

Comment:
Could we make these error assertions more explicit to what we expect to be stated?

Reply:
yeah!

Reply:
I think we could be even more explicit - e.g., instead of `parse contents of` we could say `unknown configuration elements buildpacks.url` since that is also in the error message.